amr-wind/wind_energy/actuator/FLLC.cpp

```diff
@@ -38,6 +43,64 @@ void FLLCInit(
     data.dr[0] = vs::mag(view.pos[1] - view.pos[0]) / 2.;
     data.dr[npts - 1] = vs::mag(view.pos[npts - 1] - view.pos[npts - 2]) / 2.;
 
+    //
+    //  The following code is used to create a non-uniform distribution of
+    //  points. This should follow a spacing given by epsilon/dr and values
+    //  above 1 are recommended for converged solutions.
+    //
+    if (data.nonuniform) {
+        // Non-uniform radial distribution variables
+        amrex::Real dr;
+        amrex::Real dr1;
+        amrex::Real eps1;
+        amrex::Real dr2;
+        amrex::Real eps2;
+
+        // Initialize the first location
+        data.nonuniform_r.push_back(data.r[0]);
+        while (data.nonuniform_r.back() < data.r.back()) {
+
+            amrex::Real r_ =
+                data.nonuniform_r.back(); // The latest radial position
+
+            // Interpolate the value of epsilon to the current radial location
+            eps1 = interp::linear(data.r, data.optimal_epsilon, r_);
+            dr1 = eps1 / data.eps_dr; // spacing needed to match condition
+
+            // Interpolate the value of epsilon to the next radial location
+            eps2 = interp::linear(data.r, data.optimal_epsilon, r_ + dr1);
+            dr2 = eps2 / data.eps_dr;
+
+            // This will ensure that the spacing will always meet the
+            // requirement eps/dr
+            dr = std::min(dr1, dr2);
+            AMREX_ALWAYS_ASSERT(dr > std::numeric_limits<amrex::Real>::epsilon());
```

Comment:
My change wasn't formatted!
```suggestion
            AMREX_ALWAYS_ASSERT(
                dr > std::numeric_limits<amrex::Real>::epsilon());
```